<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_Trem1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_7">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trem2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_8">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>80</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_9">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>80</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_10">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trem3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_12">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_14">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_15">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>320</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_16">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>200</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_17">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>250</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_18">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_19">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>21</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: yellow
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trem4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trem5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Trem6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: red;
font: 75 12pt &quot;Arial&quot;;
</string>
    </property>
    <property name="text">
     <string>T6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar/Retomar</string>
    </property>
   </widget>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_trilho1_5</zorder>
   <zorder>label_trilho1_6</zorder>
   <zorder>label_trilho1_7</zorder>
   <zorder>label_trilho1_8</zorder>
   <zorder>label_trilho1_9</zorder>
   <zorder>label_trilho1_10</zorder>
   <zorder>label_trilho1_11</zorder>
   <zorder>label_trilho1_12</zorder>
   <zorder>label_trilho1_14</zorder>
   <zorder>label_trilho1_15</zorder>
   <zorder>label_trilho1_16</zorder>
   <zorder>label_trilho1_17</zorder>
   <zorder>label_trilho1_18</zorder>
   <zorder>label_trilho1_19</zorder>
   <zorder>label_Trem5</zorder>
   <zorder>label_Trem6</zorder>
   <zorder>label_Trem3</zorder>
   <zorder>label_Trem1</zorder>
   <zorder>label_Trem2</zorder>
   <zorder>label_Trem4</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
